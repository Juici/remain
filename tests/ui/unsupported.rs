#[remain::check]
fn main() {
    let value = 0;

    #[sorted]
    match value {
        0..=20 => {}
        _ => {}
    }
}

#[remain::sorted]
struct TestUnnamedStruct(usize, usize, usize, usize);

#[remain::sorted]
struct TestUnitStruct;
