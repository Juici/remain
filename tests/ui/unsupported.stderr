error: unsupported by #[remain::sorted]
 --> $DIR/unsupported.rs:7:9
  |
7 |         0..=20 => {}
  |         ^^^^^^

error: expected enum, named struct, or match expression
  --> $DIR/unsupported.rs:12:1
   |
12 | #[remain::sorted]
   | ^^^^^^^^^^^^^^^^^

error: expected enum, named struct, or match expression
  --> $DIR/unsupported.rs:15:1
   |
15 | #[remain::sorted]
   | ^^^^^^^^^^^^^^^^^
